### DISCUSSION TITLE
Comnent about updates

### DISCUSSION THREAD

**Discussion by NineOutOfTen** [bromite/bromite#2535] [General]
Unfortunately, last year there were a number of occasions where you all fell behind on updates. Thankfully, it seems like you're are back on track.

As a long time user of Bromite, I hope that you all don't fall behind on them again. If I had the skill sets to aid you with your project I would. 

Thank you and I appreciate your efforts.

**Comment by RITFW** [comment]
I also came here to check if Bromite is still being maintained. And since I didn't read anything to the contrary, I assume so.

As far as I know maintainers do everything in their spare time and it is amazing what Bromite has become. A great addition to the browser landscape on Android. And it usually is on track with the Chromium version and patches - and had been for years.

I'd like to thank all the maintainers, first of all csagan5 for all the effort that was put into Bromite. You really achieved a lot with it. 

And if development is not moving fast enough, we users should ask ourselves if we have done enough, instead of complaining. Donations, contributions, etc.

**Comment by HarriBuh** [comment]
I'm worried because of the archiving of the Bromite Webview and the lack of Dev feedback since December 2022. Does the the archiving mean that the development of Bromite's Webview has been suspended?

**Comment by Dunge** [comment]
Since a week or more I have a warning in the menu asking to upgrade and leading to [this page](https://www.bromite.org/vulnerable-version) but there doesn't seem to have any release to upgrade to. Is that normal? Is that what you mean by falling behind on Chromium?

**Reply by n-buna404** [reply]
It means that bromite is lacking behind a major version of chrome currently(ie chrome 109 vs now bromite 108). So if latest version of chrome had fixed any serious security vulnerabilities the fixes are not ported to bromite yet hence using bromite now may bring risk upon users.
Users are encourage to check if there is such vulnerabilities or not before continuing using older version bromite  since the dev haven release new update and may want to switch to more updated browser for browsing activities temporarily.
